designer/jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   roots: ["<rootDir>/client"],
-  testMatch: ["<rootDir>/client/conditions/__tests__/*.jest.(ts|tsx)"],
+  testMatch: ["<rootDir>/**/__tests__/*.jest.(ts|tsx)"],
```

Comment:
I've also made this same change in my PR. Please let me know when this has been merged so I can update my branch with this code 🙏